src/SourceBuild/content/repo-projects/package-source-build.proj

```diff
@@ -7,18 +7,18 @@
     <EnableSourceControlManagerQueries>false</EnableSourceControlManagerQueries>
   </PropertyGroup>
 
+  <Import Project="$(MSBuildThisFileDirectory)installer-deps.props" />
```

Comment:
Why is the installer dependency not sufficient? Installer itself already depends on these other repos.

Reply:
See https://github.com/dotnet/installer/pull/18557#discussion_r1482034716. But also because the logic that generates derived data from `RepositoryReference` doesn't get the benefit of being able to recursively walk a dependency graph. But again, I think I can implement this a different way.